version1/04 - Web services.md

```diff
@@ -1113,6 +1113,57 @@ Content-Type: application/json
 
 # Access Control web services
 
+## Getting information about available service and explicit permissions
+
+Retrieve information about available service and explicit permissions used in managing access groups and permissions
+
+**Request**
+
+```text
+GET /rms/api/public/noark5/v1/access-group/info HTTP/1.1
+Authorization: Bearer ACCESS_TOKEN
+```
+
+**Response**
+
+```text
+Content-type: application/json
+
+{
+  "servicePermissions": [string],
+  "explicitPermissions": [string]
+}
+```
+
+The available service permissions at the time of writing this documentation are:
+* Edit list values
+* Write changelog
+* Store light PDF
+* Edit journaled
+* Dispose documents
+* Edit finalized
+* Publish documents
+* View changelog
+* Journal
+* Store documents
+* GUI administrator
+* Security administrator
+
+New service permissions may be added over time.
+
+The available explicit permissions at the time of writing this documentation are:
+* ReadThis
```

Comment:
Is there any additional documentation about what there explicit permissions mean? Same about the service permissions.

Reply:
No, not right now.

I'll add a description of this in a separate PR, if you don't mind?